install_config/deploying_cfme.adoc

```diff
@@ -0,0 +1,510 @@
+
+----
+$ oc describe pod manageiq-0
+----
+
+Readiness probes will take a while to become `Healthy` in this output. The
+initial health probes will not happen for at least eight minutes depending on
+how long it takes you to pull down the large images. CFME is a large application
+so it may take a considerable amount of time for it to deploy and be marked as
+`Healthy`.
+
+You can find which node the application is running on by checking the `oc
+describe` output, as well:
+
+----
+Successfully assigned manageiq-0 to <host|ip>
+----
+
+You can run a `docker pull` command on the node to monitor the progress of the
+image pull:
+
+----
+ifdef::openshift-origin[]
+# docker pull docker.io/manageiq/manageiq-pods:app-latest-fine
+Trying to pull repository docker.io/manageiq/manageiq-pods ...
+sha256:6c055ca9d3c65cd694d6c0e28986b5239ba56bbdf0488cccdaa283d545258f8a: Pulling from docker.io/manageiq/manageiq-pods
+Digest: sha256:6c055ca9d3c65cd694d6c0e28986b5239ba56bbdf0488cccdaa283d545258f8a
+Status: Image is up to date for docker.io/manageiq/manageiq-pods:app-latest-fine
+endif::[]
+ifdef::openshift-enterprise[]
+# docker pull registry.access.redhat.com/cloudforms45/cfme-openshift-app
+Using default tag: latest
+Trying to pull repository registry.access.redhat.com/cloudforms45/cfme-openshift-app ... 
+sha256:bc6baac5aeba5affe0bada1bfbe330cd2d58da82767d66b3fa9ab12471a1b0f5: Pulling from registry.access.redhat.com/cloudforms45/cfme-openshift-app
+
+d55ab3b04d8b: Already exists 
+b94f985aad49: Already exists 
+3cd23d7690bd: Already exists 
+Digest: sha256:bc6baac5aeba5affe0bada1bfbe330cd2d58da82767d66b3fa9ab12471a1b0f5
+Status: Image is up to date for registry.access.redhat.com/cloudforms45/cfme-openshift-app:latest
+endif::[]
+----
+
+The output above demonstrates the case where the image has been successfully
+pulled already. If the image is not completely pulled already, then you will see
+multiple progress bars detailing each image layer download status.
+
+[[deploying-cfme-opening-remote-shell]]
+=== Opening a Remote Shell to the CFME Pod
+
+You can use the `oc rsh` command to open a remote shell session to the CFME pod,
+allowing for additional inspection and progress monitoring techniques.
+
+. On your master node, switch to the *cfme* project (or whatever you named it if
+you overrode the `openshift_cfme_project` variable), and check on the pod
+states:
++
+----
+$ oc project cfme
+Now using project "cfme" on server "https://10.10.0.100:8443".
+
+$ oc get pod
+NAME                 READY     STATUS    RESTARTS   AGE
+manageiq-0           0/1       Running   0          14m
+memcached-1-3lk7g    1/1       Running   0          14m
+postgresql-1-12slb   1/1       Running   0          14m
+----
++
+Note how the *manageiq-0* pod says `0/1` under the `READY` column. After some
+time (depending on your network connection), you will be able to `oc rsh` into
+the pod to find out more of what is happening in real time.
```

Comment:
Similar to my comment above "to find out more of what is happening in real time" is a little vague. Is there something more specific to suggest?